CMakeLists.txt

```diff
@@ -564,6 +560,14 @@ if (ENABLE_ALL OR ENABLE_AZURE)
 	createExtension(AZURE-EXTENSIONS "AZURE EXTENSIONS" "This enables Azure support" "extensions/azure" "${TEST_DIR}/azure-tests")
 endif()
 
+## Add the systemd extension
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
How does this behave on distros without systemd? e.g Artix

Reply:
If the processor is used, it would fail to `dlopen("libsystemd.so.0")` during `onSchedule`

Reply:
I don't have such a distro available to me right now, so if you do and don't mind verifying that it compiles, that would be appreciated. The extension should compile without issues on such distros.

Reply:
checked it, compiles without an issue

Reply:
Awesome, thank you